codalab/lib/beam/blobstoragefilesystem.py

```diff
@@ -0,0 +1,301 @@
+#
```

Comment:
Is this copied verbatim from somewhere?  If so, we should say where it's copied from, why we can't use it as a library, and any changes we're making.